### PR TITLE
Various refactoring

### PR DESCRIPTION
**Pull request by rieder**

### CODE REVIEW

**Comment by rieder** [comment]
Mostly refactoring files with the `black` tool, but also includes some manual fixes (e.g. spelling corrections), removing obsolete syntax, and extending functionality by passing **kwargs

File: src/amuse/ic/plummer.py [hunk 1]
```diff
@@ -6,28 +6,41 @@
...
+        random_state=None,
+        random=None,
+        **kwargs
+    ):
+        self.kwargs = kwargs
         self.number_of_particles = number_of_particles
         self.convert_nbody = convert_nbody
-        self.mass_cutoff = min(mass_cutoff, self.calculate_mass_cuttof_from_radius_cutoff(radius_cutoff))
+        self.mass_cutoff = min(
+            mass_cutoff, self.calculate_mass_cuttof_from_radius_cutoff(radius_cutoff)
```
Comment (LourensVeen) [inline src/amuse/ic/plummer.py:36]: calculate_mass_cuttof -> calculate_mass_cutoff?

File: src/amuse/ic/kingmodel.py [hunk 2]
```diff
@@ -279,60 +309,65 @@ def poisson(self):
...
     def setpos(self):
         # // Obtain a random position for body b from the King profile
         # // and return the scaled potential at that location.
-        
+
         #  //  Choose radius randomly from the mass distribution.
         rno = numpy.random.uniform()
         i = int(self.NINDX * rno)
         found_index = False
-        for i1 in range(self.index[i],self.index[i+1]+2): #(i1 = self.indx[i]; i1 <= indx[i+1]+1; i1++)
```
Comment (LourensVeen) [inline src/amuse/ic/kingmodel.py:317]: Just drop the comment instead.

**Review by LourensVeen — APPROVED** [review]
Looks good, a few things to look at and maybe fix but I'm happy to merge as-is too.

File: src/amuse/ic/plummer.py (hunk 1 above)
Comment (rieder) [reply src/amuse/ic/plummer.py:36]: good catch!

**Pull request merged**